php/aplus-app/config.yaml

```diff
@@ -0,0 +1,8 @@
+framework:
+  github: aplus-framework/app
+  version: 23.3
+  website: aplus-framework.com/packages/app
+
+  bootstrap:
+    - mkdir -p storage/logs
+    - chown -R www-data:www-data storage/logs
```

Comment:
```suggestion
framework:
  github: aplus-framework/app
  version: 23.3
  website: aplus-framework.com/packages/app

  bootstrap:
    - mkdir -p storage/logs
    - chown -R www-data:www-data storage/logs
  deps:
    - libsodium
    
  modules:
    - sodium
```

Reply:
Here are the extensions we use: https://gitlab.com/aplus-framework/images/base/-/blob/3/Dockerfile#L15-33